src/ascii.rs

```diff
@@ -551,22 +515,31 @@ impl fmt::Display for IntoAsciiError {
     }
 }
 
+#[cfg(not(feature = "no_std"))]
 impl Error for IntoAsciiError {
     fn description(&self) -> &'static str {
         ERRORMSG_CHAR
     }
 }
 
 
+#[cfg(not(feature = "no_std"))]
 /// Convert `char`, `u8` and other character types to `Ascii`.
 pub trait IntoAscii : AsciiExt {
```

Comment:
When we rename `Ascii` to `AsciiChar`, should we also rename this to `IntoAsciiChar`?